packages/base-types/src/models/variableState.ts

```diff
@@ -0,0 +1,9 @@
+export type VariableValue = string | boolean | number | null;
+
+export interface Model {
+  _id: string;
+  name: string;
+  projectID: string;
+  startFrom: { diagramID: string; stepID: string } | null;
```

Comment:
lets create an interface for it

```suggestion
  interface StartFrom {
    diagramID: string; 
    stepID: string
  }

  startFrom: Nullable<StartFrom>;
```